CoastalResilienceNetwork/GeositeFramework
# Fix plugin initialization, implement single plugin mode, and support plugin ordering.

## Overview

Makes a small fix to allow plugins to load, as well as implements single plugin mode and plugin ordering.

See commit messages for details.

Connects #1129 

### Demo

![image](https://user-images.githubusercontent.com/1042475/46680282-bec70300-cbb6-11e8-9046-14294d6469cd.png)

## Testing Instructions

A few things to test:

- Load the site, and verify that the plugins you have install load and can be activated.
- Add the regional planning plugin. Verify that it and the identify point plugin are ordered correctly in the sidebar based on the `pluginOrder` setting of region.json. Reverse the order in region.json, restart the server, and verify that the plugin order is switched.
- Turn on single plugin mode in region.json and access the site. Verify that single plugin mode loads correctly.
